alf/algorithms/mcts_models.py

```diff
@@ -236,17 +269,13 @@ def recurrent_inference(self, state, action):
         Returns:
             ModelOutput: the prediction
         """
-        if not self._handle_bn:
-            dyn_state = self._dynamics_net((state.state, action))[0]
-            return self._predict(state._replace(state=dyn_state))
-        else:
+        current_steps = state.step + 1
```

Comment:
Just found that this breaks the build. Originally `state.step` is only updated for `_handle_bn` case, which means that (I could not directly point out where) `state.step` may be `()`. In that case `() + 1` will throw exception.

Reply:
Fixed